compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1614,6 +1615,19 @@ fn linker_with_args<'a, B: ArchiveBuilder<'a>>(
     let cmd = &mut *codegen_results.linker_info.to_linker(base_cmd, &sess, flavor);
     let link_output_kind = link_output_kind(sess, crate_type);
 
+    if let LinkerFlavor::Lld(LldFlavor::Gcc) = flavor {
+        let tools_path =
+            sess.host_filesearch(PathKind::All).get_tools_search_paths(crt_objects_fallback);
+        // FIXME handle rust-lld not found
+        let lld_path = tools_path.into_iter().find(|p| p.join("rust-lld").exists()).unwrap();
+        cmd.arg({
+            let mut arg = OsString::from("-B");
+            arg.push(lld_path);
+            arg
+        });
+        debug!("{:?}", cmd.cmd().get_args());
+    }
```

Comment:
Should this be part of `linker_info.to_linker()`?

Reply:
sorry, i don't understand your question. what should be part of `linker_info.to_linker()`? the debug output?

Reply:
This whole block to add the `-B` arg to the linker command.